DEPENDENCIES

```diff
@@ -124,11 +124,35 @@ Prerequisites for building the OpenDataPlane (ODP) API
    $ ./configure --with-crypto=armv8crypto
    $ make
 
-3.4 Netmap packet I/O support (optional)
+3.4 Multi-buffer Crypto for IPsec Library (optional)
+
+   Multi-Buffer Crypto for IPsec Library is a set of functions that
+   implement authentication and encryption processing for IPsec, these functions
+   take advantage of SIMD instructions and process multiple buffers in parallel
+   to improve performance.
```

Comment:
The "process multiple buffers in parallel" part does not seem to be true for the way this code uses the library, so it is a bit misleading to mention that property here.

Reply:
Thanks for your comments, removed this part.